### ISSUE TITLE
Confusing UI during address entry

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#2295]
This is not really a bug, but if you enter your address and uncheck "Set as Default Billing/Set as default Shipping" you are left with a screen where it's unclear what you are supposed to do next.
Only after clicking around did I figure out that I needed to click on the addresses again to use them for this order.

![cart_checkout](https://cloud.githubusercontent.com/assets/72819/26134640/88f3f7d0-3ae2-11e7-88d2-2064c8c9f012.png)

**Comment by spencern** [comment]
This address book UI was the single biggest issue GetOutfitted customers had during checkout. Many customers figured it out, but a non trivial number also had a lot of issues with this, especially when their billing address and shipping address were different. Intuitive move to have different billing address than shipping is to click the "edit/pencil" icon on the right beside the billing address, but that changes both.

**Comment by rymorgan** [comment]
Adding to the design project. We need to design out some other options for ways we can handle the address book UI.

**Comment by aaronjudd** [comment]
@impactmass @spencern status update? this has been rewritten since this issue was created, and should/could have been addressed?

**Comment by zenweasel** [comment]
@rymorgan This has had the design label removed, is this still under design? Should we classify this as a bug?

**Comment by rymorgan** [comment]
@zenweasel Since I can't have my own board for managing issues when they need design because of limitations with zenhub, I've been playing with different ways to address using labels. What I've come up with is labeled design means, it needs design. Design pipeline means we've identified it as something we are going to be working on in the near future. Design-complete and put back in the pipeline we pulled it from means it's ready to go. Re-labeled this design. This is probably a UI bug too.
